streams/src/test/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImplTest.java

```diff
@@ -45,7 +46,14 @@ public void shouldNotHaveNullReducerOnReduce() throws Exception {
 
     @Test(expected = NullPointerException.class)
     public void shouldNotHaveNullStoreNameOnReduce() throws Exception {
-        groupedStream.reduce(MockReducer.STRING_ADDER, null);
+        String storeName = null;
+        groupedStream.reduce(MockReducer.STRING_ADDER, storeName);
```

Comment:
I separated it intentionally, as there are two `reduce` methods of a particular class and when second argument is `null` it gives compile time error (`null` does not have type). So, instead of giving `null` argument, I assigned `null` to String not to get compile time error.